opentelemetry-zipkin/src/lib.rs

```diff
@@ -140,6 +177,7 @@ impl Default for ZipkinPipelineBuilder {
             service_addr: None,
             collector_endpoint: DEFAULT_COLLECTOR_ENDPOINT.to_string(),
             trace_config: None,
+            client: Box::new(reqwest::blocking::Client::new()),
```

Comment:
May want to use the optional version that I had and use the features to set defaults if present, currently this will not compile without the `reqwest` feature.

Reply:
I think I made the `reqwest` as default feature to make sure builder will always have some kind of http client. But think it again, it still can be broken when user override the default features. So it make sense to have an option here. Will fix it.